packages/read-model/src/Product/Listed/ListedProductViewElasticFacade.php

```diff
@@ -269,12 +270,16 @@ protected function createFromArray(array $productsArray): array
         $listedProductViews = [];
         foreach ($productsArray as $productArray) {
             $productId = $productArray['id'];
-            $listedProductViews[$productId] = $this->listedProductViewFactory->createFromArray(
-                $productArray,
-                $imageViews[$productId],
-                $this->productActionViewFactory->createFromArray($productArray),
-                $this->currentCustomerUser->getPricingGroup()
-            );
+            try {
+                $listedProductViews[$productId] = $this->listedProductViewFactory->createFromArray(
+                    $productArray,
+                    $imageViews[$productId],
+                    $this->productActionViewFactory->createFromArray($productArray),
+                    $this->currentCustomerUser->getPricingGroup()
+                );
+            } catch (NoProductPriceForPricingGroupException $exception) {
+                continue;
```

Comment:
I got feedback on this line – silently dropped exceptions can be confusing when you're solving some unexpected behavior. 
I'm not really sure how to deal with this information.